### ISSUE TITLE
Can't use icons

### ISSUE THREAD

**Issue by note89** [react-native-community/React-Native-Elements#25]
Hi i have used several elements but when i try to use Icons like this 

```
import {  Icon } from 'react-native-elements'

<Icon
  name='rowing' />
```

i am met by this screen 
![image](https://cloud.githubusercontent.com/assets/6696963/18555317/c1c288b4-7b67-11e6-9cd4-94419a9a358b.png)

**Comment by fossage** [comment]
Have you installed the latest version?  It looks like the Icon component was just added yesterday.

**Comment by dabit3** [comment]
@fossage @note89 apologies, and yes, this is fixed in version 4.4 and later. Sorry about the inconvenience!

**Comment by note89** [comment]
Tnx :) i will upgrade !
